unstructured/ingest/cli/cmds/azure.py

```diff
@@ -7,43 +7,57 @@
 from unstructured.ingest.cli.interfaces import (
     CliConfig,
 )
+from unstructured.ingest.connector.azure import (
+    AzureAccessConfig,
+    AzureWriteTextConfig,
+    SimpleAzureBlobStorageConfig,
+)
 
 CMD_NAME = "azure"
 
 
 @dataclass
-class AzureCliConfig(CliConfig):
-    account_id: t.Optional[str] = None
-    account_name: t.Optional[str] = None
-    connection_string: t.Optional[str] = None
-
+class AzureCliConfig(AzureAccessConfig, CliConfig):
     @staticmethod
     def get_cli_options() -> t.List[click.Option]:
         options = [
             click.Option(
                 ["--account-key"],
                 default=None,
-                help="Azure Blob Storage or DataLake account key (not required if "
-                "`azure_account_name` is public).",
+                help="The storage account key. This is used for shared key "
+                "authentication. If any of account key, sas token or "
+                "client_id is specified, anonymous access will be used.",
```

Comment:
Is this a typo, it says anonymous access will be used when credentials are provided